docs/README.md

```diff
@@ -3,3 +3,20 @@
 ## Browser support
 
 We align our browser support targets with that of Wagtail. Have a look at the [official documentation](http://docs.wagtail.io/en/latest/contributing/developing.html).
+
+## Python/Django/Wagtail support
+
+Supporting Python versions as defined in `setup.py` classifiers.
+
+Supporting Django versions as defined in the Django project [supported versions](https://www.djangoproject.com/download/#supported-versions).
+
+Supporting Wagtail versions from `1.7`.
```

Comment:
`1.12` has just been release so I'll use that. However my understanding is that there are never 2 LTS versions at the same time? I'll remove `1.8` from the test matrix for now but we can add it again if I'm wrong.